dialog_lib/db/session.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+import sqlalchemy as sa
+from sqlalchemy.orm import Session
+
+def get_session():
+    engine = sa.create_engine(os.environ.get("DATABASE_URL"))
+    session = Session(engine)
+    yield session
+    session.close()
```

Comment:
```suggestion
def get_session():
    engine = sa.create_engine(os.environ.get("DATABASE_URL"))
    with Session(engine) as session:
        yield session
```

Reply:
Already made a change using the contextlib. Can you re-review this PR?